### PR TITLE
usability changes for check_wfc

### PR DESCRIPTION
**Pull request by grahamlopez**
There are no functionality changes here.

I found out after looking at the source of check_wfc that I had slightly forgotten how the arguments to check_wfc work, and had slipped at some point into using incorrectly by asking for the different components as arguments without '-f'. However, it happily accepted this and printed "All checking pass" for J2, but since I had gotten into the habit of only looking for "All checking pass" and the messages about what is being built/checked scroll too quickly off the top of my screen, I failed to notice that it wasn't checking what I asked for.

This seemed like a mistake someone else could fall for eventually, so I added a little bit more stringent checking to the existing arguments, as well as an option for help output that informs about all of the options. 

I also added the component name to the final line of output.

### CODE REVIEW

File: src/Drivers/check_wfc.cpp [hunk 1]
```diff
@@ -68,41 +89,51 @@ int main(int argc, char **argv)
 
   char *g_opt_arg;
   int opt;
-  while ((opt = getopt(argc, argv, "hvVs:g:i:r:f:")) != -1)
+  while(optind < argc)
```
Comment (ye-luo) [inline src/Drivers/check_wfc.cpp]: I feel the old code is fine. We only need to see null in the wfc_name default and the later check will catch the issue. I think all the other defaults are fine.

**Comment by ye-luo** [comment]
I also often make such mistakes...

File: src/Drivers/check_wfc.cpp (hunk 1 above)
Comment (grahamlopez) [reply src/Drivers/check_wfc.cpp]: this code allows us to keep a default for quick usage, and disallows something like `check_wfc J1`, which currently will check J2 instead without explanation. It also catches any other unknown arguments with a help message.

What is the disadvantage of having this?

Comment (ye-luo) [reply src/Drivers/check_wfc.cpp]: Now I see what you mean.

**Pull request merged**